src/main/java/Calculator.java

```diff
@@ -0,0 +1,20 @@
+public class Calculator {
+
```

Comment:
as for branch name feature-calculator

Usually for feautre most common format:
feature/....

so 
feature/calculator
feature/add-calc-functionality
feature/ABC-563  (if there are JIRA number)
feature/add-book-service

You could do nothing in this case, just get know for future